project/src/main/webapp/event-details.js

```diff
@@ -64,7 +82,8 @@ function getListItemForOpportunity(
 
   // If the user is logged in and the current user is the event host,
   // then show the edit link for the volunteering opportunity.
-  if (loginState === 'LOGGED_IN' && !currentUser.localeCompare(eventHost)) {
+  if (loginStatus.loginState === 'LOGGED_IN' &&
```

Comment:
(this line updates if you use the shared.js function, as it abstracts away the login status value and just returns true/false)

Reply:
Ack, will use in other cases but I actually need both here because I am using event host.

Reply:
ah yeah - I just meant the first part - if you used the function in shared.js, the first part will just be something like if(isLoggedIn() && ...

Reply:
I see - done.